main2.py

```diff
@@ -0,0 +1,99 @@
+import random
+used_letters = []
+user_word = []
+word_list = []
+lifes = 5
+
+
+def decor(func):
+    def wrapp():
+        print("-----------------------")
+        func()
+    return wrapp
+
+
+def list_creator(word_list):

```

Comment:
Modyfikacja parametru to anty-pattern, powinieneś zwrócić tutaj arraya a nie go modyfikować